bin/spice/cmd/trace.go

```diff
@@ -26,23 +26,33 @@ import (
 )
 
 var (
-	// If `--last` was requested
-	isLast bool
-
 	// The id of the trace to provide
 	id string
 
 	// The trace_id of the trace to provide
 	trace_id string
 )
 
+var supported_trace_tasks = []string{"ai_chat", "ai_completion", "sql_query", "nsql"}
+
+func isValidTraceTask(task string) bool {
+	for _, supported_task := range supported_trace_tasks {
```

Comment:
Also considered adding the following to check for `tool_use`, decided to list explicitly to avoid potential risks w/ sql injection/etc

```
		// support for tool_use::sample_data, etc
		if strings.HasPrefix(task, "tool_use::") {
			return true
		}
```